recipyCmd/cmd_annotate.py

```diff
@@ -0,0 +1,43 @@
+import click
+from tinydb import where
+from recipyCmd.templating import render_run_template
+from recipyCmd.recipycmd import pass_config
+from recipyCommon.utils import get_run
+
+
+@click.command('annotate', short_help='Add a note to the latest run.')
+@click.option('--id', '-i', help='Tag run which id starts with TEXT.')
+@pass_config
+def cmd(config, id):
+    """Add a note to the latest run using default text editor.
+     You can also tag older runs using --id.
+
+    \b
```

Comment:
Hope you are feeling better and thanks for lookign through the code. I should have the refactoring wrapped up by tonight and tested, I will keep the search as it is and only iron out the gremlins.

The \b here is to prevent the re-wrappping of the lines that follow. It will be deleted from the help message dispalyed. You can see the full explanation here http://click.pocoo.org/5/documentation/